Worldmap/WorldMap.htm

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE HTML>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <script src="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.js"></script>
+    <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.css" />
+    <style>
+      html, body {
+        height: 100%;
+        padding: 0;
+        margin: 0;
+      }
+      #map {
+        /* configure the size of the map */
+        width: 100%;
+        height: 100%;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="map"></div>
+    <script>
+      // adapted from https://switch2osm.org/using-tiles/getting-started-with-leaflet/ 
+      // initialize Leaflet
+      var map = L.map('map').setView({lon: 0, lat: 0}, 2);
+
+      // add the OpenStreetMap tiles
+      L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+        maxZoom: 19,
+        attribution: '&copy; <a href="https://openstreetmap.org/copyright">OpenStreetMap contributors</a>'
+      }).addTo(map);
+
+      // show the scale bar on the lower left corner
+      L.control.scale().addTo(map);
+
+      // show a marker on the map
+// hier weitere Lokalitäten ergänzen
+      L.marker({lon: 16.373056, lat: 48.208333}).bindPopup('Wien<br><a href="https://de.wikipedia.org/wiki/Wien">Wikipeadia</a>').addTo(map);
+      L.marker({lon: 149.116667, lat: -35.3}).bindPopup('Canberra<br><a href="https://de.wikipedia.org/wiki/Canberra">Wikipeadia</a>').addTo(map);
+    </script>
+  </body>
+</html>
```

Comment:
Duplicate of #20 

Diese Datei ist bereits Bestandteil es oben genanten Pull requests. 
Sie sollte daher an dieser Stelle nicht erneut aufgenommen werden.

Reply:
Mir ist unklar, wie diese Datei nochmal in den pull-request geraten ist. Wie kann ich sie wieder entfernen?

Reply:
Wenn du wieder das Leerzeichen einfügst müsste die Änderung eigentlich verschwinden.

Reply:
Das sollte die Datei aus Git wieder entfernen: 
`git rm -r --cached Worldmap/`